BraveShared/BraveStrings.swift

```diff
@@ -1418,6 +1418,210 @@ extension Strings {
+                              value: "Open a New Browser Tab",
+                              comment: "")
+        
+        public static let activityTypeNewPrivateTabTitle =
+            NSLocalizedString("shortcuts.activityTypeNewPrivateTabTitle",
+                              bundle: .braveShared,
+                              value: "Open a New Private Browser Tab",
+                              comment: "")
+        
+        public static let activityTypeClearHistoryTitle =
+            NSLocalizedString("shortcuts.activityTypeClearHistoryTitle",
+                              bundle: .braveShared,
+                              value: "Clear Brave Browsing History",
+                              comment: "")
+        
+        public static let activityTypeEnableVPNTitle =
+            NSLocalizedString("shortcuts.activityTypeEnableVPNTitle",
+                              bundle: .braveShared,
+                              value: "Open Brave Browser and Enable VPN",
+                              comment: "")
+        
+        public static let activityTypeOpenBraveTodayTitle =
+            NSLocalizedString("shortcuts.activityTypeOpenBraveTodayTitle",
+                              bundle: .braveShared,
+                              value: "Clear Brave Browsing History",
+                              comment: "")
+        
+        public static let activityTypeOpenPlaylistTitle =
+            NSLocalizedString("shortcuts.activityTypeOpenPlaylistTitle",
+                              bundle: .braveShared,
+                              value: "Open Playlist",
+                              comment: "")
+        
+        public static let activityTypeTabDescription =
+            NSLocalizedString("shortcuts.activityTypeTabDescription",
+                              bundle: .braveShared,
+                              value: "Start Searching the Web Securely with Brave",
+                              comment: "")
+        
+        public static let activityTypeClearHistoryDescription =
+            NSLocalizedString("shortcuts.activityTypeClearHistoryDescription",
+                              bundle: .braveShared,
+                              value: "Open Browser in a New Tab and Delete All Private Browser History Data",
+                              comment: "")
+        
+        public static let activityTypeEnableVPNDescription =
+            NSLocalizedString("shortcuts.activityTypeEnableVPNDescription",
+                              bundle: .braveShared,
+                              value: "Open Browser in a New Tab and Enable VPN",
+                              comment: "")
+        
+        public static let activityTypeBraveTodayDescription =
+            NSLocalizedString("shortcuts.activityTypeBraveTodayDescription",
+                              bundle: .braveShared,
+                              value: "Open Brave Today and Check Today's Top Stories",
```

Comment:
This has been renamed to `Brave News`

Reply:
Fix at https://github.com/brave/brave-ios/pull/3579/commits/6094980027fe2d3ea410dd134ff093f8876e7a3c